backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5621500_sys_gh12323_Adding_EORI_Number.sql

```diff
@@ -0,0 +1,72 @@
+DROP FUNCTION IF EXISTS de_metas_endcustomer_fresh_reports.docs_purchase_order_description(IN record_id   numeric,
+                                                                                           IN ad_language Character Varying(6))
```

Comment:
And the migration scripts for the case when we adding the functions that we are using in reports, shall be in: https://github.com/metasfresh/metasfresh/tree/9b07692c02cf5041238b7a5300edf47433ffdb9c/backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/75-de.metas.fresh.reports